internal/persistence/sql/migrations/uuid_mapping.go

```diff
@@ -0,0 +1,56 @@
+package migrations
+
+import (
+	"context"
+
+	"github.com/gobuffalo/pop/v6"
+	"github.com/gofrs/uuid"
+
+	"github.com/ory/keto/internal/persistence/sql"
+)
+
+type (
+	toUUIDMappingMigrator struct {
+		d dependencies
+	}
+)
+
+// NewToUUIDMappingMigrator creates a new UUID mapping migrator.
+func NewToUUIDMappingMigrator(d dependencies) *toUUIDMappingMigrator {
+	return &toUUIDMappingMigrator{d: d}
+}
+
+// MigrateUUIDMappings migrates to UUID-mapped subject IDs for all relation
+// tuples in the database.
+func (m *toUUIDMappingMigrator) MigrateUUIDMappings(ctx context.Context) error {
+	p, ok := m.d.Persister().(*sql.Persister)
+	if !ok {
+		panic("got unexpected persister")
+	}
+
+	return p.Transaction(ctx, func(ctx context.Context, c *pop.Connection) error {
+		var relationTuples []*sql.RelationTuple
+
+		if err := p.Connection(ctx).All(&relationTuples); err != nil {
```

Comment:
Getting all will probably not work for bigger databases. Please add pagination to this. You can use keyset pagination https://www.cockroachlabs.com/docs/stable/pagination.html#keyset-pagination for that. Should be the fastest one.

Reply:
I now added pagination with `(pop.Query).Paginate`, which seems to be the cross-db way of doing things.